tardis/tests/tests_slow/conftest.py

```diff
@@ -19,6 +22,12 @@ def pytest_configure(config):
         config.option.integration_tests_config = yaml.load(
             open(integration_tests_configpath))
 
+        if "generate_reference" not in config.option.integration_tests_config:
+            config.option.generate_reference['generate_reference'] = None
```

Comment:
If `generate_reference` is not at all mentioned, it throws `KeyError`. Just to avoid `KeyError` in the fixture down below. I can do this in fixture itself, but I had to make this comparison for setting `dokureport_active` anyhow.